zerver/views/webhooks/github.py

```diff
@@ -136,7 +141,7 @@ def api_github_landing(request, user_profile, event=REQ(),
                        exclude_commits=REQ(converter=flexible_boolean, default=False),
                        emphasize_branch_in_topic=REQ(converter=flexible_boolean, default=False),
                        ):
-
+    # type: (HttpRequest, UserProfile, text_type, Disct[text_type, Any], text_type, text_type, int, text_type, text_type, bool, bool, bool, bool) -> HttpResponse
```

Comment:
`HttpRequest` is not defined. You'll have to import it otherwise mypy will give a 'not defined' error.